features/definitions/resolvers-steps.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const should = require('should'),
```

Comment:
In our coding standard we declare one `const` per line.  
So it's easier to copy/paste